### ISSUE TITLE
Validate Ocean assets throw ValueError

### ISSUE THREAD

**Issue by mariacarmina** [oceanprotocol/ocean.py#1268]
For `test_ocean_assets_validate`, it fails with the following error message:

```console
ValueError: DDO has validation errors: {'errors': {'id': 'did is not valid for chain Id and nft address', 'nftAddress': 'nftAddress is missing or invalid.'}}
```

Full stack trace:
```python

self = <ocean_lib.ocean.ocean_assets.OceanAssets object at 0x7fb870a85940>
ddo = <ocean_lib.assets.ddo.DDO object at 0x7fb870ad0dc0>

    @enforce_types
    def validate(self, ddo: DDO) -> Tuple[bool, list]:
        """
        Validate that the ddo is ok to be stored in aquarius.
    
        :param ddo: DDO.
        :return: (bool, list) list of errors, empty if valid
        """
        # Validation by Aquarius
        validation_result, validation_errors = self._aquarius.validate_ddo(ddo)
        if not validation_result:
            msg = f"DDO has validation errors: {validation_errors}"
            logger.error(msg)
>           raise ValueError(msg)
E           ValueError: DDO has validation errors: {'errors': {'id': 'did is not valid for chain Id and nft address', 'nftAddress': 'nftAddress is missing or invalid.'}}

ocean_lib/ocean/ocean_assets.py:75: ValueError
```
Error found [here](https://github.com/oceanprotocol/ocean.py/actions/runs/3858229599/jobs/6576515586)

**Comment by mariacarmina** [comment]
Fix should be to provide different sample DDO schema since aquarius has changed.

**Comment by AnaLoznianu** [comment]
Closed by #1267
